src/condition/size_condition.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+from src.condition.i_condition import ICondition
+
+class SizeCondition(ICondition):
```

Comment:
Size is not the same as length. This is a length condition rather than size - size refers to the byte size rather than length.